test/test262/expectations/index.js

```diff
@@ -85,7 +100,10 @@ module.exports = {
       // ES2018 invalid escapes in template literals: https://github.com/tc39/ecma262/pull/773
       'language/expressions/tagged-template/invalid-escape-sequences.js',
 
-      ...regexpXfail,
+      'annexB/language/literals/regexp/class-escape.js', // /\c0/ is invalid in es2016
```

Comment:
Is this new behavior in ES2018? The test predates it, so that would be surprising.

If the test is wrong, we should file a bug against test262. But how sure are you that the test is wrong, rather than the parser? All browser JS engines accept `/\c0/`.

Reply:
Browsers accept it because they implement more than es2016. In es2016, there is a lookahead preventing the IdentityEscape from allowing a `\c`, and the control escape with decimal digits is only allowed in classes. In es2018 they add a reverse solidus to the [spec](https://tc39.github.io/ecma262/#prod-annexB-ExtendedAtom), see the difference in the [es2016 spec](https://www.ecma-international.org/ecma-262/7.0/#prod-annexB-ExtendedAtom).

Reply:
The ES2016 spec has a production going `ExtendedAtom` -> `ExtendedPatternCharacter` -> `\`, which looks to me like it should allow `\c`.

The change you're point it looks like it was in https://github.com/tc39/ecma262/pull/910, which as far as I can tell is just a refactoring and bugfix preventing `/\/` from being legal.

Reply:
Yep, you're right. Addressed here: https://github.com/shapesecurity/shift-regexp-acceptor-js/pull/10